lib/sparrow/h2_worker/config.ex

```diff
@@ -54,15 +63,21 @@ defmodule Sparrow.H2Worker.Config do
         authentication,
         tls_options \\ [],
         ping_interval \\ 5_000,
-        reconnect_attempts \\ 3
+        reconnect_attempts \\ 3,
+        backoff_base \\ 2,
+        backoff_initial_delay \\ 100,
```

Comment:
Let's change the input here to a map - this way, optional values can be truly optional and can be defined in one place (a default map that we can merge input into).